hellomama_registration/utils.py

```diff
@@ -92,6 +93,24 @@ def patch_identity(identity, data):
     return r.json()
 
 
+def search_optouts(params=None):
```

Comment:
Not strictly necessary, but if we have `**params` here, then we can call this function `search_optouts(reason='other', request_source='ussd')` instead of `search_optouts(params={'reason': "other", 'request_source': "ussd"})`